packages/openapi-ts/src/plugins/@hey-api/sdk/config.ts

```diff
@@ -82,6 +83,13 @@ export const defaultConfig: HeyApiSdkPlugin['Config'] = {
     } else {
       plugin.config.instance = false;
     }
+
+    // Set default classNameBuilder based on client type
+    if (plugin.config.classNameBuilder === '{{name}}') {
+      if (plugin.config.client === '@hey-api/client-angular') {
+        plugin.config.classNameBuilder = '{{name}}Service';
+      }
+    }
```

Comment:
Why?

Reply:
Yes I'm also conflicted about this, for now the `Service` suffix is still widely accepted by Angular devs, but they too would prefer to move away from this pattern